tests/test_misc.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+import hashlib
+
+import pytest
+import requests
+import requests_mock
+
+from sentinelsat import SentinelAPI, SentinelAPIError
+from sentinelsat.sentinel import _parse_opensearch_response
+
+
+@pytest.mark.fast
+def test_checksumming_progressbars(capsys, fixture_path):
+    api = SentinelAPI("mock_user", "mock_password")
+    testfile_md5 = hashlib.md5()
+    true_path = fixture_path("expected_search_footprints_s1.geojson")
+    with open(true_path, "rb") as testfile:
+        testfile_md5.update(testfile.read())
+        real_md5 = testfile_md5.hexdigest()
+
+    assert api._md5_compare(true_path, real_md5) is True
+    out, err = capsys.readouterr()
+    assert "checksumming" in err
+    api = SentinelAPI("mock_user", "mock_password", show_progressbars=False)
+    assert api._md5_compare(fixture_path("map.geojson"), real_md5) is False
+    out, err = capsys.readouterr()
+    assert out == ""
+    assert "checksumming" not in err
+
+
+@pytest.mark.vcr
+@pytest.mark.skip(reason="Cannot mock since VCR.py has issues with Unicode request bodies.")
```

Comment:
Can you point to docs or an issue regarding this? I couldn't find anything in the vcr.py docs. If this will not be implemented into vcr.py in the future this test would be on "permanent skip" and thus could be removed.

Reply:
There's a small bug in vcr.py which prevents it from playing back responses with Unicode. I submitted a patch for it at https://github.com/kevin1024/vcrpy/pull/386. Should be merged any day now...
Should have included a note about this at the unit test, I'll fix that.